x-pack/legacy/plugins/reporting/export_types/csv/server/create_job.ts

```diff
@@ -4,12 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { oncePerServer } from '../../../server/lib/once_per_server';
 import { cryptoFactory } from '../../../server/lib/crypto';
-import { ConditionalHeaders, CreateJobFactory, ServerFacade, RequestFacade } from '../../../types';
-import { JobParamsDiscoverCsv, ESQueueCreateJobFnDiscoverCsv } from '../types';
+import { ConditionalHeaders, ServerFacade, RequestFacade } from '../../../types';
+import { ESQueueCreateJobFnDiscoverCsv, JobParamsDiscoverCsv } from '../types';
 
-function createJobFn(server: ServerFacade) {
+export function createJobFactory(server: ServerFacade): ESQueueCreateJobFnDiscoverCsv {
```

Comment:
Oops, there is another PR that does these changes: https://github.com/elastic/kibana/pull/49426/files#diff-e6c74eb5d977f06c7f116aa268248b9cR11

Reply:
Let's go with this one and I'll close the other

Reply:
Inspired by @joelgriffith's PR, I think it would be better to do this like:
```
export const createJobFactory: ESQueueCreateJobFn = function createJobFn(server: ServerFacade) {
```

That's what I was originally looking for in my changes, but couldn't get the JS right.